CONTRIBUTIONS.md

```diff
@@ -0,0 +1,63 @@
+### Guidelines for Pull Requests and Releases (Web3 1.x)
+
+This document provides some ground rules for contributors (including the maintainer(s) of
+the project) about how to make, review and publish changes to 1.x. The most basic requirement is
+that **Web3 not break**.
+
+## Pull Requests for substantive changes (e.g. everything except comments and docs)
+
+1. Any PR that introduces a logic change should include tests. (In many cases, the tests will take
+more time to write than the actual code).
+
+2. All PRs should sit for 48 hours with the `pleasereview` tag in order to garner feedback.
+
+3. No PR should be merged until it has been reviewed, passes CI, and all reviews' comments are
+addressed.
+
+4. PRs should have a narrow, well-defined focus and make the smallest set of changes possible to achieve
+their goal.
+
+5. Given the choice between a conservative change that mostly works and an adventurous change which
+seems better but introduces uncertainty - prefer the conservative change.
+
```

Comment:
I think we can add here a small sentence about the importance of well-described PR's.

Reply:
Agree - I added this to the end of number 4 because it seems like those things go together.